docs/operator-manual/applicationset/Generators-Matrix.md

```diff
@@ -22,8 +22,8 @@ As an example, imagine that we have two clusters:
 
 And our application YAMLs are defined in a Git repository:
 
-- Argo Workflows controller (examples/git-generator-directory/cluster-addons/argo-workflows)
-- Prometheus operator (/examples/git-generator-directory/cluster-addons/prometheus-operator)
+- [Argo Workflows controller](https://github.com/argoproj/argo-cd/tree/master/applicationset/examples/git-generator-directory/cluster-addons/argo-workflows)
```

Comment:
mkdocs doesnt like relative paths so we cannot do `../../../applicationset/examples/git-generator-directory/cluster-addons/prometheus-operator`